lib/docdog/accounts/accounts.ex

```diff
@@ -103,7 +103,7 @@ defmodule Docdog.Accounts do
   end
 
   def find_or_create(%Auth{} = auth) do
-    query = from(u in User, where: u.username == ^auth.info.nickname)
+    query = from(u in User, where: u.email == ^auth.info.email)
```

Comment:
I am not quite sure about this: does Github API always provide an email in the payload? Or if user decides to make his email private we will get `nil` here?

Reply:
We need to use `github_uid` here. Please, read my comment in issue #25.